src/rules/label-has-for.js

```diff
@@ -17,35 +17,41 @@ const validateNesting = (node, allowChildren) =>
     if (child.type === "VElement") {
       return (
         !isHiddenFromScreenReader(child) &&
-        (controlTypes.includes(getElementType(child)) ||
+        (controlTypes
+          .concat(controlComponents)
+          .includes(getElementType(child)) ||
           validateNesting(child, allowChildren))
```

Comment:
don't we need to pass the controlComponents to this calling of the funciton also?